core/db/entities.py

```diff
@@ -1,19 +1,40 @@
 import uuid  # noqa
 
-from sqlalchemy import Column, String
+import sqlalchemy
 
 from core.db import db
 from core.db.types import GUID
 
 
+class Organisation(db.Model):
+    __tablename__ = "organisation_dim"
+
+    organisation_id = sqlalchemy.Column(GUID(), default=uuid.uuid4, primary_key=True)
+    organisation_name = sqlalchemy.Column(
+        sqlalchemy.String, nullable=False, unique=True
+    )
+    geography = sqlalchemy.Column(sqlalchemy.String, nullable=True)
```

Comment:
Are we sure `geography` is a string? Can we confirm with Maeve?

Reply:
@nsnape seems like there is some uncertainty around this, see https://communities-govuk.slack.com/archives/C04UVF4U01X/p1681803446408639?thread_ts=1681734853.042499&cid=C04UVF4U01X can we add a TODO so we don't forget to review this?

Reply:
FYI @mgoodwin-version1